python/cudf/cudf/tests/conftest.py

```diff
@@ -196,6 +196,26 @@ def set_decomp_env_vars(monkeypatch, request):
         yield
 
 
+@pytest.fixture(
+    params=[
+        "min",
+        "max",
+        "sum",
+        "product",
+        "quantile",
+        "all",
+        "any",
+        "std",
+        "var",
+        "median",
+        "kurtosis",
+        "skew",
```

Comment:
We could consider making the reductions list public: https://github.com/rapidsai/cudf/blob/branch-25.10/python/cudf/cudf/core/mixins/reductions.py#L11. We're only testing a subset here, though. Not sure if that is intentional.